README.md

```diff
@@ -3,6 +3,41 @@ The open data portal is a serverless data store for Stanford datasets. We store
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+## Data Collection
+
+## Dataset Submission 
+
+## Open Data Pipeline
+
+## Frontend Structure
+We have the following types of pages/components on our website.
+### Landing Page
+Our home page gives an overview of the website, highlighting some datasets and data-viz stories.
+
+### About Us
+This page provides some background on the team working on this website.
+
+### Datasets
+This page contains a comprehensive list of all the datasets on our site. Users can also search directly for a dataset or narrow down their choices by category. 
+
+### Dataset Details
+There is a dataset details page for each dataset. These pages contain more detailed information about the dataset, and provide links to download the dataset from the original source and from our cleaned CSV version. We have also added dataset preview and links to stories that use the dataset.
+
+### Contribute
+We have a form that allows users of our site to leave us messages. The form responses are then automatically sent to our email address by using the [emailjs](https://www.emailjs.com/) service. 
+
+Additionally, we embedded a [Box](https://www.box.com/home) folder onto this page. This allows users to submit dataset file directly to us, while ensuring that they cannot access (and possibly delete) other files within our folder. 
+
+Note: We intially tried to allow submission of files through our form, but there is a limit on the size of data that passes through the emailjs service.
+
+### Other
+All of these pages share the same CSS template (styles.css) and all of the photos used on our site are under the static folder.
+
+## Feature Spotlights
+
+## Contact Us!
+If you're interesting in creating your own open data project, feel free to contact us via email, stanfordopendataproject@gmail.com.
```

Comment:
Do we want to use a @stanforddaily.com email instead? I don't remember if data viz already has an email at this domain, @arjunramani3 ?